libs/Onboarding/src/Onboarding/Accounts/AccountsService.cpp

```diff
@@ -331,20 +323,22 @@ QJsonObject AccountsService::prepareAccountSettingsJsonObject(const GeneratedAcc
             {"eip1581-address", account.derivedAccounts.eip1581.address},
             {"dapps-address", account.derivedAccounts.defaultWallet.address},
             {"wallet-root-address", account.derivedAccounts.walletRoot.address},
-            {"preview-privacy", true},
+            {"preview-privacy?", true},
```

Comment:
Question mark?

Reply:
It seems it is a requirement so I bring it back :). See [status-go](https://github.com/status-im/status-go/blob/f6c9ec7838b91aba325351bda2903f25a8ba5547/multiaccounts/settings/structs.go#L145)